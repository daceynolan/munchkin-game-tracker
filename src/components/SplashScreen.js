import React from "react";

import Header from "../components/Header";
import Button from "./Button";
import MainCharacter from "../images/main-character.png";
import "./SplashScreen.css";

const SplashScreen = () => {
  return (
    <div className="splash-screen">
      <Header className="splash-screen__header" />
      <img
        className="splash-screen__character-image"
        src={MainCharacter}
        alt="Munchkin character holding chainsaw"
      />
      <Button>New Game</Button>
    </div>
  );
};

export default SplashScreen;
