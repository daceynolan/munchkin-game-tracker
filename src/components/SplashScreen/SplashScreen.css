.splash-screen {
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
}

.splash-screen__character-image {
  height: 300px;
  width: auto;
  margin-bottom: 20px;
}
