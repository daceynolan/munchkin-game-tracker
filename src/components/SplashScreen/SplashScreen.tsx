import React from "react";
import { Link } from "react-router-dom";

import "./splashScreen.css";
import Button from "components/Button";
import MainCharacter from "images/main-character.png";

const SplashScreen = () => {
  return (
    <div className="splash-screen">
      <img
        className="splash-screen__character-image"
        src={MainCharacter}
        alt="Munchkin character holding chainsaw"
      />
      {/* 
// @ts-ignore */}
      <Button as={Link} to="/configure">
        New Game
      </Button>
    </div>
  );
};

export default SplashScreen;
