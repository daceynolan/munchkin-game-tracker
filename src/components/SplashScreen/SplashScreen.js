import React from "react";
import { Link } from "react-router-dom";

import Header from "../Header";
import Button from "../Button";
import MainCharacter from "../../images/main-character.png";
import "./SplashScreen.css";

const SplashScreen = () => {
  return (
    <div className="splash-screen">
      <Header className="splash-screen__header" />
      <img
        className="splash-screen__character-image"
        src={MainCharacter}
        alt="Munchkin character holding chainsaw"
      />
      <Button as={Link} to="/configure">
        New Game
      </Button>
    </div>
  );
};

export default SplashScreen;
