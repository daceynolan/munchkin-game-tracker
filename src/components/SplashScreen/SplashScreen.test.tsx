import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

import SplashScreen from "./";

describe("<SplashScreen/>", () => {
  test("renders without crashing", () => {
    const { container } = render(
      <MemoryRouter>
        <SplashScreen />
      </MemoryRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
