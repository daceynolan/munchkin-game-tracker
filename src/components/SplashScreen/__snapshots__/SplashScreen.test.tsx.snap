// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SplashScreen/> renders without crashing 1`] = `
<div
  class="splash-screen"
>
  <img
    alt="Munchkin character holding chainsaw"
    class="splash-screen__character-image"
    src="main-character.png"
  />
  <a
    class="button button--default"
    href="/configure"
  >
    New Game
  </a>
</div>
`;
